src/components/Routing/SegmentPage.js

```diff
@@ -58,7 +61,7 @@ export const List = styled.div`
 	flex-direction: column;
 `;
 
-const FilteredLink = ({ active, ...props }) => <Link {...props} />;
+const FilteredLink = ({ active, ...props }) => (props.to ? <Link {...props} /> : <div {...props} />);
```

Comment:
What is the purpose of a div with the props?

Reply:
If Section is not disabled it is rendered as LINK, if it is disabled, it rendered as general <div>:
![image](https://user-images.githubusercontent.com/6649972/107045721-83609180-67ce-11eb-8007-9d27feac6439.png)